test/BountyRegistry.test.js

```diff
@@ -759,4 +686,632 @@ contract('BountyRegistry', function ([owner, user0, user1, user2, expert0, exper
       arbiter0Candidates[2].should.equal(user2);
     });
   });
+
+  describe('payouts', function() {
+    it('should refund bounty amount and bounty fee to ambassador if there are no assertions or votes', async function() {
+      let amount = ether(10);
+      let bid = ether(20);
+      let tx = await postBounty(this.token, this.bountyregistry, user0, amount, IPFS_README, 2, 10);
+      let guid = tx.logs[0].args.guid;
+
+      // wait for assertion period to pass
+      await advanceBlocks(10);
+
+      // wait for reveal period to pass
+      await advanceBlocks(ASSERTION_REVEAL_WINDOW);
+
+      await advanceBlocks(ARBITER_VOTE_WINDOW);
+
+      await settleBounty(this.bountyregistry, user0, guid);
+
+      let ambassadorBalance = await this.token.balanceOf(user0);
+      // init - bountyFee
+      ambassadorBalance.should.be.bignumber.equal(ether(STARTING_EXPERT_BALANCE));
+    });
+
+    it('should refund bounty amount to ambassador if there are no assertions', async function() {
+      let amount = ether(10);
+      let bid = ether(20);
+      let tx = await postBounty(this.token, this.bountyregistry, user0, amount, IPFS_README, 2, 10);
+      let guid = tx.logs[0].args.guid;
+
+      // wait for assertion period to pass
+      await advanceBlocks(10);
+
+      // wait for reveal period to pass
+      await advanceBlocks(ASSERTION_REVEAL_WINDOW);
+
+      await voteOnBounty(this.bountyregistry, arbiter0, guid, 0x1);
+      await voteOnBounty(this.bountyregistry, arbiter1, guid, 0x1);
+      await voteOnBounty(this.bountyregistry, arbiter2, guid, 0x3);
+
+      await advanceBlocks(ARBITER_VOTE_WINDOW);
+
+      await settleBounty(this.bountyregistry, user0, guid);
+      await settleBounty(this.bountyregistry, arbiter0, guid);
+
+      let bounty = await this.bountyregistry.bountiesByGuid(guid);
+      let selected = bounty[6];
+      selected.should.not.be.bignumber.equal(0);
+
+      if (selected != arbiter0) {
+        await settleBounty(this.bountyregistry, selected, guid);
+      }
+
+      let ambassadorBalance = await this.token.balanceOf(user0);
+      // init - bounty fee
+      ambassadorBalance.should.be.bignumber.equal(ether(STARTING_EXPERT_BALANCE).sub(BOUNTY_FEE));
+
+      let arbiterBalance = await this.token.balanceOf(selected);
+      // init + bounty fee
+      arbiterBalance.should.be.bignumber.equal(ether(STARTING_ARBITER_BALANCE).add(BOUNTY_FEE));
+    });
+
+    it('should refund bounty fee to ambassador if there are no votes', async function() {
```

Comment:
Is this misnamed? Ambassador doesn't get its bounty fee back for this case.

Reply:
I added that per ticket #115 @ceseale